ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -0,0 +1,98 @@
+@{
+    var notificationId = ViewData["NotificationId"];
+}
+
+<div class="notification-overview-type-and-edit-container">
+    <h3 class="notification-details-type"> Clinical details </h3>
+    <a href="./Edit/ClinicalDetails?id=@notificationId" class="notification-edit-link"> Edit </a>
+</div> 
+<div class="notification-overview-details-container">
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Sites of Disease </div>
+            <div class="empty-cell-min-height"> @Model.SitesOfDisease </div>

```

Comment:
empty-cell-min-height is used to create some space between fields when data is missing - especially useful while we have a lot of placeholders

Reply:
Sounds good - let's put a comment on the class definition in the css file.
Also, as its applied to all cells not just the empty ones, I'd suggest we don't use `empty`  in its name